src/webpack/webpack.config.js

```diff
@@ -0,0 +1,141 @@
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const rtl = require('postcss-rtl');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CleanPlugin = require('clean-webpack-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const aggregateTranslations = require('../../scripts/aggregate-translations/aggregate-translations');
+const merge = require('webpack-merge');
```

Comment:
nice! I saw this a while ago and it seemed like we should follow the recommended webpack pattern.